src/commands/spawn.ts

```diff
@@ -21,6 +21,7 @@ export default class Spawn extends Command {
     grace: Flags.integer({ char: 'g', description: 'optional grace period (in seconds) to wait after last connection is closed before shutting down container' }),
     port: Flags.integer({ char: 'p', description: 'optional port for jamsocket to proxy requests to (default is 8080)' }),
```

Comment:
Not related to this PR, but perhaps we should also hide the `port` flag from docs / help here and in the create spawn token command.